nixos/modules/security/pam.nix

```diff
@@ -533,6 +552,9 @@ let
               optionalString cfg.gnupg.enable ''
                 auth optional ${pkgs.pam_gnupg}/lib/security/pam_gnupg.so ${optionalString cfg.gnupg.storeOnly " store-only"}
               '' +
+              optionalString cfg.failDelay.enable ''
+                auth optional ${pkgs.pam}/lib/security/pam_faildelay.so delay=${cfg.failDelay.delay}
```

Comment:
Did not test, shall we use `${toString cfg.failDelay.delay}`?

Reply:
Thanks, I forget this.